docs/overview/ai-incident-assistant.mdx

```diff
@@ -0,0 +1,23 @@
+---
+title: "AI Incident Assistant"
+---
+
+<Tip>
+Keep Cloud: ✅ <br/>
+Keep Enterprise On-Premises: ✅ <br/>
+Keep Open Source: ✅ 🤔 (experimental)
```

Comment:
I don't like those icons too much because it kinda signals like we don't know something but lgtm'ing